app/views/widgets/home_page/_emergency_alert.html

```diff
@@ -0,0 +1,20 @@
+<div class="emergency-alert">
+  <span class="close-alert-trigger">
+    <svg class="icon-close" version="1.1" xmlns="http://www.w3.org/2000/svg" width="13" height="13" viewBox="0 0 512 512">
+      <title>Close</title>
+      <path d="M446.627 110.627l-45.254-45.254-145.373 145.372-145.373-145.372-45.254 45.253 145.373 145.374-145.373 145.373 45.254 45.254 145.373-145.373 145.373 145.373 45.254-45.255-145.372-145.372z"></path>
+    </svg>
+  </span>
+  <div class="alert-text">
+    <svg id="icon-warning" xmlns="http://www.w3.org/2000/svg" width="32" height="32" viewBox="0 0 32 32">
+      <title>warning</title>
```

Comment:
Should this also be capitalized to be consistent with the `title` above?

Reply:
sure